compiler/modules/front.syntax/ast.bal

```diff
@@ -36,6 +36,7 @@ public type FunctionDefn record {|
     Visibility vis;
     FunctionTypeDesc typeDesc;
     string[] paramNames;
+    Position[] paramPos;
```

Comment:
We have too many inconsistent ways of doing the position for a name:

- name + namePos
- paramNames + paramPos arrays
- varName + varPos
- `[string, Position]` identity


"identity" is worse than "name" or "identifier".